.github/workflows/codecov.yml

```diff
@@ -14,45 +14,49 @@ jobs:
       LLVM_PROFILE_FILE: "nomos-node-%p-%m.profraw"
     runs-on: self-hosted
     steps:
-      - uses: actions/checkout@v4
-        with:
-          submodules: true
-      - uses: arduino/setup-protoc@v3
+      - name: Checkout repository
+        uses: actions/checkout@v4
+      - name: Install protoc compiler
```

Comment:
We can remove this step, protobufs were removed from DA.

Reply:
For all jobs?

Reply:
Yes, no use of protobufs in any part of the node at the moment.

Reply:
Good to know! https://github.com/logos-co/nomos-node/pull/1058/commits/6d09c4880d77a8959c7fbba814e6435ed66c6ec5